.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
   - jdk: openjdk7
   - jdk: openjdk8
   - jdk: oraclejdk8
+  - jdk: openjdk11
+  - jdk: oraclejdk11
```

Comment:
:question: Why 11 and not any versions before (or after)? We don't have any tests in our regression suit for this version so this might go wrong. I'm not sure if allowing this straight away is the best way of extending the support..

Reply:
> Why 11 and not any versions before (or after)?

9 & 10 are not long term support according to https://en.wikipedia.org/wiki/Java_version_history - in fact they are already not supported.

I'd certainly be open to adding 12, but thought one version at a time for now. 11 is often the version that comes by default e.g. with the maven docker image, so is the one I want to work for my own purposes right now.

We should remove 7 at some point as it's no longer supported upstream, but @peterschrammel said it's still wanted right now.

> We don't have any tests in our regression suit [sic] for this version so this might go wrong.

Yes, it might go wrong. That's why we need to test it in CI. The current deeptest-utils test suite does work with this, and that's all I am implying here. If there are issues in deeptest-utils related to 11 support, we should add tests that expose them, and use travis' allow_failures feature.

Adding this is not meant to imply that we officially support Java 11, it's just intended to reduce the chance that people commit things that go backwards on that front. 

deeptest-utils as a standalone project could support Java 11 without implying that the rest of our projects do. This would be useful because people could have a project that supports Java 8 & 11, and be able to run the generated tests on both, even if they can only generate them on 8. Verifying that behaviour of your project doesn't change when you upgrade the Java platform is one of the most useful features of an automatically generated test suite.